best_practices/code-analysis/default.eslintrc.yml

```diff
@@ -70,14 +80,13 @@ rules:
     no-with: 2
     quotes: [2, "double"]
     radix: 2
-    require-jsdoc: 2
+    require-jsdoc: 0
     semi: 2
     semi-spacing: [2, {before: false, after: true}]
-    space-after-keywords: [2, "always"]
+    keyword-spacing: [2, {before: true, after: true}]
```

Comment:
Got it! From [`space-after-keywords` page](https://eslint.org/docs/rules/space-after-keywords):

> This rule was removed in ESLint v2.0 and replaced by the keyword-spacing rule.